INSTALL.txt

```diff
@@ -168,9 +190,8 @@ MS compilers
 If you are familiar with MS tools, that's obviously the easiest path, and
 the compilers are hopefully more mature (although in my experience, they
 are quite fragile, and often segfault on invalid C code). The main drawback
-is that no fortran compiler + MS compiler combination has been tested -
-mingw-w64 gfortran + MS compiler does not work at all (it is unclear
-whether it ever will).
+is that mingw-w64 gfortran + MSVC does not work at all (it is unclear
```

Comment:
Oh, maybe we're talking past each other. Surely being able to use gfortran
to build Python extensions on windows at all requires that gfortran + MSVC
be working at the technical level, b/c that's what you end up with after
Python loads your extension. Getting a build system to actually use MSVC
for C code while using gfortran for fortran is another matter of course...?

On Sat, May 23, 2015 at 1:40 PM, Ralf Gommers <notifications@github.com>
wrote:

> In INSTALL.txt
> <https://github.com/numpy/numpy/pull/5912#discussion_r30948079>:
>
> > @@ -168,9 +190,8 @@ MS compilers
> >  If you are familiar with MS tools, that's obviously the easiest path, and
> >  the compilers are hopefully more mature (although in my experience, they
> >  are quite fragile, and often segfault on invalid C code). The main drawback
> > -is that no fortran compiler + MS compiler combination has been tested -
> > -mingw-w64 gfortran + MS compiler does not work at all (it is unclear
> > -whether it ever will).
> > +is that mingw-w64 gfortran + MSVC does not work at all (it is unclear
>
> I haven't tried recently, but why would it? Carl's toolchain is all of
> MinGW, I don't think MSVC + gfortran was even a goal (it would be very nice
> to have though).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/numpy/numpy/pull/5912/files#r30948079>.
>



-- 
Nathaniel J. Smith -- http://vorpus.org